_data/pages/es/downloads.yml

```diff
@@ -1,9 +1,14 @@
 banner:
-  title: Download
+  title: Downloads
 
 intro:
-  title: Get started
-  ext_content: We provide Docker images, RPM files, Debian packages, and more.
+  title: Ready to get started?
+  ext_content: We provide Docker images, RPM files, Debian packages and more. Jump to <a href="https://opendistro.github.io/for-elasticsearch-docs/">getting started guide</a> to dive deep in to Open distro for Elasticsearch.
```

Comment:
I think we can omit the second sentence here, but if you want to include it, I recommend something like:

`See the <a href="https://opendistro.github.io/for-elasticsearch-docs/">documentation</a> for a deep dive into Open Distro for Elasticsearch.`

Reply:
I have made the suggested changes.